pkgs/jupiter-hw-support/theme.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, callPackage
+, xorg
+}:
+
+let
+  src = callPackage ./src.nix { };
+in
+stdenv.mkDerivation {
+  pname = "steamdeck-theme";
+
+  inherit src;
+  inherit (src) version;
+
+  nativeBuildInputs = [
+    xorg.xcursorgen
+  ];
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share
+    cp -r usr/share/icons $out/share
+    cp -r usr/share/plymouth $out/share
+    cp -r usr/share/steamos $out/share
+
+    sed -i "s|/usr/|$out/|g" $out/share/plymouth/themes/steamos/steamos.plymouth
+
+    pushd $out/share/steamos
+    xcursorgen steamos-cursor-config $out/share/icons/steam/cursors/default
+    popd
+  '';
+
+  meta = with lib; {
+    description = "Steam Deck themes";
+    license = licenses.unfreeRedistributable;
```

Comment:
Why is this one `unfreeRedistributable`?

Given the previous comment about the license, I would have assumed this was MIT too, that only the firmware bits were unfree. Is there something I'm missing about the license here?

Reply:
I used it since I think the Steam Deck logo and other assets might be considered unfree. Maybe we can reach out to Valve for clarification since they are apparently slapping `license=('MIT')` to all of their PKGBUILDs and it may be inaccurate.

Also I forgot to add the `overlay.nix` changes. Let me fix that.

Reply:
I don't know if there's any avenue other than bugging `@ \ Plagman2` on twitter or `@ \ Plagman` on github. Tried on twitter once for the USB shenanigans, but never got a reply. Understandable since he must receive a **lot** of unneeded mentions.